packages/@adobe/spectrum-css-temp/components/tray/index.css

```diff
@@ -94,3 +94,10 @@
     border-radius: var(--spectrum-tray-border-radius);
   }
 }
+
+@media (forced-colors: active) {
+  .spectrum-Tray {
```

Comment:
i think you're missing an `overflow: hidden` (which should be general, not behind the media query) like the other popovers above?